> **Gene expression, chromosome heterogeneity and the fast-X effect in mammals**
> 
> Linh-Phuong Nguyen, Nicolas Galtier, Benoit Nabholz
> 
> [_Biol Lett 2015 Feb;11(2):20150010.  doi: 10.1098/rsbl.2015.0010_](https://pubmed.ncbi.nlm.nih.gov/25716091/)



| Attribute      | Value | 
| :---        |    :----   | 
| Number of MSA      | 4981       | 
| MSA Format	      | `FASTA` | 
| Stored as	      | `.gz` one file per alignment       | 
| Aligners used     | Sourced from a database of MSAs       | 
| Filtering strategy     | OrthoMAM v6, `GBlocks`    |
| Tree	      | Yes, single species tree (`tree.nwk`) | 
| Sequences/MSA      | 6 [5:6]       | 
| Codons/MSA      | 511 [154:1898]       | 
| Tree length/MSA      | 0.21 [0.09:0.50]       | 
| Gaps/codon column     | 0.34 [0.0:1.17]       | 
| Gaps/1000 codons sequence   | 58.1 [0.0:203.0]      | 
| Taxonomic range      | Mammals       | 
| Selection analyses run      | Counting-based lineage dN/dS estimation   |


Abstract
--------
The higher rate of non-synonymous over synonymous substitutions (dN/dS) of the X chromosome compared with autosomes is often interpreted as a consequence of X hemizygosity. However, other factors, such as gene expression, are also known to vary between X and autosomes. Analysing 4800 orthologues in six mammals, we found that gene expression levels, associated with GC content, fully account for the variation in dN/dS between X and autosomes with no detectable effect of hemizygosity. We also report an extensive variance in dN/dS and gene expression between autosomes.