> **Estimates of Positive Darwinian Selection Are Inflated by Errors in Sequencing, Annotation, and Alignment**
> 
> Adrian Schneider, Alexander Souvorov, Niv Sabath, Giddy Landan, Gaston H. Gonnet, and Dan Graur
> 
> _Genome Biol Evol. 2009; 1: 114–118_



| Attribute      | Value | 
| :---        |    :----   | 
| Number of MSA      | 9404       | 
| MSA Format	      | `PHYLIP` / `paml` | 
| Stored as	      | `.bz2` tarball, one file per alignment       | 
| Aligners used     | Darwin multiple sequence alignment package       | 
| Filtering strategy     | Manual, based on ad hoc criteria       |
| Tree	      | Yes, using a single species tree, stored in `tree.nwk`  | 
| Sequences/MSA      | 7       | 
| Codons/MSA      | 469 [120:1850]       | 
| Tree length/MSA      | 0.41 [0.17:0.86]       | 
| Gaps/codon column     | 0.18 [0.0:1.0]       | 
| Gaps/1000 codons sequence   | 25.3 [0.0:141.9]       | 
| Taxonomic range      | Mammals       | 
| Selection analyses run      | PAML branch-site models (seven terminal lineages)    |

Abstract
--------
Published estimates of the proportion of positively selected genes (PSGs) in human vary over three orders of magnitude. In mammals, estimates of the proportion of PSGs cover an even wider range of values. We used 2,980 orthologous protein-coding genes from human, chimpanzee, macaque, dog, cow, rat, and mouse as well as an established phylogenetic topology to infer the fraction of PSGs in all seven terminal branches. The inferred fraction of PSGs ranged from 0.9% in human through 17.5% in macaque to 23.3% in dog. We found three factors that influence the fraction of genes that exhibit telltale signs of positive selection: the quality of the sequence, the degree of misannotation, and ambiguities in the multiple sequence alignment. The inferred fraction of PSGs in sequences that are deficient in all three criteria of coverage, annotation, and alignment is 7.2 times higher than that in genes with high trace sequencing coverage, “known” annotation status, and perfect alignment scores. We conclude that some estimates on the prevalence of positive Darwinian selection in the literature may be inflated and should be treated with caution.